helm/kubernetes-coredns-chart/templates/configmap.yaml

```diff
@@ -29,7 +29,7 @@ data:
         prometheus :{{ .Values.ports.prometheus }}
 
         {{- if .Values.configmap.forward }}
-        forward . {{ range (.Values.configmap.forward | trimAll "\n " |  split "\n") }}{{ . }}{{ end }} /etc/resolv.conf {
+        forward . {{ range (.Values.configmap.forward | trimAll "\n " |  splitList "\n") }}{{ . }}{{ end }} /etc/resolv.conf {
```

Comment:
> You can add additional forward entries by adding a each as a line to the forward field of the user ConfigMap. They will be selected in sequential order.

As per the docs, I think we need to use `splitList` to preserve the order.

https://docs.giantswarm.io/guides/advanced-coredns-configuration/#additional-forwards